### PR TITLE
OKTA-490569 add smart card idp

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add Smart Card IdP to the "Add an Identity Provider" section
- **Is this PR related to a Monolith release?** No

### Preview:
- https://626c447b9549d4009c027b48--reverent-murdock-829d24.netlify.app/docs/guides/add-an-external-idp/smartcard/

### Resolves:

* [OKTA-490569](https://oktainc.atlassian.net/browse/OKTA-490569)

### CODE REVIEW

**Comment by vanngo-okta** [comment]
Lucas Gomes (CIAM Technical Marketing Manager), who originally requested this addition, approved the changes.

**Comment by vanngo-okta** [comment]
John Baldo and Megha Rastogi approved via Slack.

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/smartcard/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Add a Smart Card Identity Provider
+meta:
+  - name: description
+    content: Okta supports smart cards with a x.509 compliant digital certificate, such as a PIV card, as a primary authentication factor to sign in to Okta.
+---
+
+## Smart Card
+
+Okta supports smart cards with an x.509 compliant digital certificate, such as a personal identity verification card, as a primary authentication factor to sign in to Okta. To configure this feature in Okta, you need to add Smart Card as an external Identity Provider.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/smartcard/index.md:10]:
```suggestion
Okta supports smart cards with an X.509 compliant digital certificate, such as a personal identity verification card, as a primary authentication factor to sign in to Okta. To configure this feature in Okta, you need to add Smart Card as an external Identity Provider.
```

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
Looks good except for one feedback item.

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/smartcard/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/smartcard/index.md]: Committed your suggestion. Thanks!

**Review by robertnicolo-okta — APPROVED** [review]
lgtm

**Pull request merged**